packages/utils/typescript/lib/utils/get-config-path.js

```diff
@@ -22,6 +22,8 @@ module.exports = (dir, { filename = DEFAULT_TS_CONFIG_FILENAME, ancestorsLookup
   if (!configFilePath || ancestorsLookup) {
     return configFilePath;
   }
+  
+  const configFileAbsolutePath = path.resolve(configFilePath);
```

Comment:
```suggestion

  // resolve path again because ts.findConfigFile() returns a POSIX path that breaks on Windows
  const configFileAbsolutePath = path.resolve(configFilePath);
```